hri_safety_sense/src/MsgHandler.h

```diff
@@ -18,7 +18,6 @@
 /**
```

Comment:
If we do keep this class/file (see above), I'd suggest renaming to `MsgHandler.hpp`.

Reply:
7de7162